src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -764,11 +765,12 @@ class FlaxBertPreTrainedModel(FlaxPreTrainedModel):
 
     config_class = BertConfig
     base_model_prefix = "bert"
-    module_class: nn.Module = None
+    module_class = None
```

Comment:
`module_class` is still an `nn.Module` no? (wrapped inside the parent dataclass `FlaxBertPreTrainedModel`)

Reply:
It is. However, I you add a type annotation it will be interpreted as a dataclass field instead of a class variable. See updated code for a possible solution.